packages/admin-next/dashboard/src/routes/orders/order-detail/components/order-summary-section/order-summary-section.tsx

```diff
@@ -406,6 +406,17 @@ const ReturnBreakdown = ({
 const Total = ({ order }: { order: AdminOrder }) => {
   const { t } = useTranslation()
 
+  const totalPaid = order.payment_collections.reduce(
```

Comment:
thought: Wonder if we should be getting this calculated from the backend? Is this not part of the order totals decoration?

Reply:
as in, setting this on the order totals?

Reply:
https://github.com/medusajs/medusa/pull/8591

I adjusted it to only add the transaction on payment capture, and not after the authorization.